nxdrive/gui/folders_dialog.py

```diff
@@ -3,6 +3,9 @@
 from pathlib import Path
 from typing import TYPE_CHECKING, Dict, List, Optional, Union
 
+from PyQt5 import QtCore
+from PyQt5.QtGui import QCursor, QRegExpValidator
```

Comment:
See the line `from ..qt.imports import ...`. Qt imports are now dissalowed and must be made from only one file.